### PR TITLE
Making Menu APIs Public

### PR DESCRIPTION
**Pull request by meethjain**
Making the following API's public 
- setUpViews
- setNavBarMenu
- showActionMenu
This is an ask from the Product as they want to increase usage of the API by 3rd Party App Developers

Verified tests are passing locally by running `yarn test`

Queries
- Is there anything we need to consider from Desktop as the ask is from Mobile Side

### CODE REVIEW

**Comment by MarcinJuraszek** [comment]
Wouldn't we want to move the files to `public` folder too?

**Comment by meethjain** [comment]
@MarcinJuraszek I've moved the menus.ts file from private to public folder. Any other file that should be moved?

**Comment by meethjain** [comment]
>

**Comment by meethjain** [comment]
@MarcinJuraszek @ydogandjiev Can you please help review this PR?

File: src/private/privateAPIs.ts [hunk 1]
```diff
@@ -10,14 +10,14 @@ import {
...
-import { menus } from './menus';
+
 import { registerHandler } from '../internal/handlers';
 import { GlobalVars } from '../internal/globalVars';
 import { ErrorCode, SdkError } from '../public/interfaces';
 import { getUserJoinedTeamsSupportedAndroidClientVersion } from '../internal/constants';
 
 export function initializePrivateApis(): void {
-  menus.initialize();
+  //Keeping this API for any future usage, wherein privateAPIs need to be initialized
```
Comment (ydogandjiev) [inline src/private/privateAPIs.ts:20]: NIT: Add a space after the "//"

**Review by ydogandjiev — APPROVED** [review]

**Review by MarcinJuraszek — APPROVED** [review]

**Pull request merged**